scripts/merge-and-build.sh

```diff
@@ -114,6 +139,8 @@ else
   fi
 fi # End check if we are master
 
+# TODO: Origin already vendors the web console. Should we stop updating it separately for OSE?
```

Comment:
Sure -- before we do the rebase to `origin/master` we can do a rebase onto ourselves -- so we can make the ordering of commits correct (i.e. the last four commits are as we want above)